src/tests/ftest/util/server_utils.py

```diff
@@ -521,6 +519,20 @@ def start(self):
 
     def stop(self):
         """Stop the server through the runner."""
+        self.log.info("<SERVER> Printing daos_admin stack traces")
+
+        prefix = "/var/tmp"
+        corefile_ss = "core.daos_admin"
+        gdb_cmd = "gdb -ex bt -ex quit"
+
+        bash_cmd = "for p in `ls %s`; do echo $p; [[ \"$p\" == *\"%s\"* ]] && %s %s/$p; done".format{
+            prefix, corefile_ss, gdb_cmd, prefix}
+
+        results = pcmd(self._hosts, bash_cmd, verbose)
+
```

Comment:
This should already be getting done for you by the test framework: https://github.com/daos-stack/daos/blob/55ae4d9cfcbf74bb7ddb60f79e558f6dd494f7bb/src/tests/ftest/launch.py#L1832-L1959

Are you finding a case where it's not being done?